imageBuilder.go

```diff
@@ -11,6 +12,21 @@ func main() {
 	// Unmarshal the yaml configuration file and generate a dockerfile.
 	err := configurations.WriteDockerfile()
 	if err != nil {
-		log.Fatalf("error while writing dockerfile %v", err)
+		log.Fatalf("error while writing dockerfile: %v", err)
+	}
+
+	authData, err := builder.GetAuthData("docker-auth.yaml")
+	if err != nil {
+		log.Fatalf("error while reading authetication details: %v", err)
+	}
+
+	err = builder.BuildImage(*authData)
+	if err != nil {
+		log.Fatalf("error while building image for code runner: %v", err)
+	}
+
+	err = builder.PushImageToHub(*authData)
```

Comment:
provide command-line options that can be used to inform whether the image needs to be pushed or not.

Reply:
Okay